FhirToCdm/Microsoft.Health.Fhir.Transformation.Core/FhirElementTabularTransformer.cs

```diff
@@ -70,7 +70,7 @@ private IEnumerable<(string name, object valueObj, object typeObj)> ToTabularInt
                     ElementNode valueNode = fhirRoot;
                     if (!string.IsNullOrEmpty(column.FhirExpression))
                     {
-                        var valueFromExpression = fhirRoot.Select(column.FhirExpression).FirstOrDefault();
+                        var valueFromExpression = fhirRoot?.Select(column.FhirExpression).FirstOrDefault();
```

Comment:
Does this mean the NDJson file is empty? Or exist an empty line?

Reply:
it means the root element is null, sample here: Patient.name== null and try to get Patient.name.(fhir expression) would fail.

Reply:
Maybe add UTs for this behavior.